docs/build/web.md

```diff
@@ -11,9 +11,11 @@ redirect_from: /docs/how-to/build/web
 
 There are 2 steps to build ONNX Runtime Web:
 
-- build ONNX Runtime for WebAssembly
-  - or skip and download a pre-built artifacts
-- build onnxruntime-web (NPM package)
+- Obtaining ONNX Runtime WebAssembly artifacts - can be done by - 
+  - Building ONNX Runtime for WebAssembly 
+  - Download the pre-built artifacts
```

Comment:
add links to this 2 options

Reply:
specifically, maybe we need a few explanations of "Download the pre-built artifacts" and point users to look at the section "Build onnxruntime-web (NPM package)" directly.

Reply:
Good idea! I changed a few things to make it work, now we have a link